tests/src/JIT/Methodical/Arrays/misc/arrres.cs

```diff
@@ -62,52 +62,52 @@ private static void Test2()
                 s_arr[i].CheckValid();
                 s_arr[i] = null;
             }
-            GC.Collect();
-            GC.WaitForPendingFinalizers();
-            GC.Collect();
         }
         [System.Runtime.CompilerServices.MethodImplAttribute(System.Runtime.CompilerServices.MethodImplOptions.NoInlining)]
         private static void Test3()
         {
+            GC.Collect();
+            GC.WaitForPendingFinalizers();
+            GC.Collect();
             for (int i = 0; i < 50; i++)
```

Comment:
I'm just making the minimal changes to make the test more correct (and leaving the basic structure intact when it's not impacted by the GC tracking changes); I have no idea why it was written this way and frankly I don't feel comfortable changing it without a specific reason and checking with whoever owns this. There are other tests similar to this one and my suggestion would be to change all of them separately (or leave them) but not conflate such a thing with this PR. Hope that makes sense but if not just let me know.